cmd/mongodb-reader/main.go

```diff
@@ -33,46 +33,46 @@ import (
 )
 
 const (
-	defThingsURL     = "localhost:8181"
-	defLogLevel      = "error"
-	defPort          = "8180"
-	defDBName        = "mainflux"
-	defDBHost        = "localhost"
-	defDBPort        = "27017"
-	defClientTLS     = "false"
-	defCACerts       = ""
-	defServerCert    = ""
-	defServerKey     = ""
-	defJaegerURL     = ""
-	defThingsTimeout = "1" // in seconds
-
-	envThingsURL     = "MF_THINGS_URL"
-	envLogLevel      = "MF_MONGO_READER_LOG_LEVEL"
-	envPort          = "MF_MONGO_READER_PORT"
-	envDBName        = "MF_MONGO_READER_DB_NAME"
-	envDBHost        = "MF_MONGO_READER_DB_HOST"
-	envDBPort        = "MF_MONGO_READER_DB_PORT"
-	envClientTLS     = "MF_MONGO_READER_CLIENT_TLS"
-	envCACerts       = "MF_MONGO_READER_CA_CERTS"
-	envServerCert    = "MF_MONGO_READER_SERVER_CERT"
-	envServerKey     = "MF_MONGO_READER_SERVER_KEY"
-	envJaegerURL     = "MF_JAEGER_URL"
-	envThingsTimeout = "MF_MONGO_READER_THINGS_TIMEOUT"
+	defLogLevel          = "error"
+	defPort              = "8180"
+	defDBName            = "messages"
+	defDBHost            = "localhost"
+	defDBPort            = "27017"
+	defClientTLS         = "false"
+	defCACerts           = ""
+	defServerCert        = ""
+	defServerKey         = ""
+	defJaegerURL         = ""
+	defThingsAuthURL     = "localhost:8181"
+	defThingsAuthTimeout = "1" // in seconds
+
+	envLogLevel          = "MF_MONGO_READER_LOG_LEVEL"
+	envPort              = "MF_MONGO_READER_PORT"
+	envDBName            = "MF_MONGO_READER_DB_NAME"
```

Comment:
If we're going to use `DB` instead of `DB_NAME` for database names, this should be applied to all the services, including all the readers and writers. Both `DB` and `DB_NAME` are OK, as long as we're consistent.